src/main/java/open/schoolmanagement/time/calendartaskprojectservice/domain/calendar/Constants.java

```diff
@@ -0,0 +1,39 @@
+/*
+   Copyright 2018 Open School Management
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+package open.schoolmanagement.time.calendartaskprojectservice.domain.calendar;
+
+public interface Constants {
+
+    public static String BAWUE = "BW";
+    public static String BAY = "BY";
+    public static String BER = "BE";
+    public static String BRA = "BB";
+    public static String BREM = "HB";
+    public static String HHAM = "HH";
+    public static String HES = "HE";
+    public static String MECPOM = "MV";
+    public static String NIESA = "NI";
+    public static String NOWEST = "NW";
+    public static String RHEINPFALZ = "RP";
+    public static String SAAR = "SL";
+    public static String SAX = "SN";
+    public static String SAXSTOP = "ST";
+    public static String SCHLESWIG = "SH";
+    public static String THUER = "TH";
+
+
+    public static long DEFAULT_COURSE_DURATION = 3000000; //50 MIN
```

Comment:
Wieso 50 Minuten? Im Saarland sind es 45 Minuten, in der Pfalz 60, ...

Reply:
In der mir bekannten Schule werden die 5 min Pause in die Stunde eingerechnet.